### PR TITLE
[Transformers] Support load mode from HF Hub when use Neural Speed

### PR DESCRIPTION
**Pull request by zhentaoyu**
## Type of Change

feature or bug fix or documentation or others
API changed or not

## Description

detail description 
JIRA ticket: xxx

## Expected Behavior & Potential Risk

the expected behavior that triggered by this PR 

## How has this PR been tested?

how to reproduce the test (including hardware information)

## Dependency Change?

any library dependency introduced or removed

### CODE REVIEW

**Review by a32543254 — APPROVED** [review]
LGTM

**Review by kevinintel — APPROVED** [review]

File: examples/huggingface/neural_speed/run_accuracy.py [hunk 1]
```diff
@@ -19,15 +19,15 @@
     parser = argparse.ArgumentParser(description="Evaluate diff for a model")
     parser.add_argument('--model_name', type=str, default="~/Llama-2-7b-chat-hf", help="path to model")
     parser.add_argument('--tasks', type=str, default="lambada_openai")
-    parser.add_argument('--model_format', type=str, default="runtime")
+    parser.add_argument('--model_format', type=str, default="neural_speed")
```
Comment (a32543254) [inline examples/huggingface/neural_speed/run_accuracy.py:22]: please also change example readme

Comment (zhentaoyu) [reply examples/huggingface/neural_speed/run_accuracy.py:22]: I have already checked it and it does not need to be modified.

**Comment by VincyZhang** [comment]
https://inteltf-jenk.sh.intel.com/job/nlp-toolkit-validation-top-mr-extension/1493/

**Comment by VincyZhang** [comment]
https://inteltf-jenk.sh.intel.com/job/nlp-toolkit-validation-top-mr-extension/1494/

**Review by changwangss — APPROVED** [review]

**Comment by VincyZhang** [comment]
https://inteltf-jenk.sh.intel.com/job/nlp-toolkit-validation-top-mr-extension/1497/

**Comment by VincyZhang** [comment]
https://inteltf-jenk.sh.intel.com/job/nlp-toolkit-validation-top-mr-extension/1499/

**Comment by VincyZhang** [comment]
https://inteltf-jenk.sh.intel.com/job/nlp-toolkit-validation-top-mr-extension/1500/

**Comment by VincyZhang** [comment]
https://inteltf-jenk.sh.intel.com/job/nlp_toolkit_optimize_validation_localtest/6014/
https://inteltf-jenk.sh.intel.com/job/nlp_toolkit_optimize_validation_localtest/6015/

**Comment by VincyZhang** [comment]
final test: https://inteltf-jenk.sh.intel.com/job/nlp-toolkit-validation-top-mr-extension/1501/

**Comment by VincyZhang** [comment]
https://inteltf-jenk.sh.intel.com/job/nlp-toolkit-validation-top-mr-extension/1502/

**Comment by VincyZhang** [comment]
https://inteltf-jenk.sh.intel.com/job/nlp_toolkit_optimize_validation_localtest/6030/

**Pull request merged**